modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -13,7 +13,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: <storage-class-name> <1>
-provisioner: kubernetes.io/azure-disk
+provisioner: disk.csi.azure.com
 volumeBindingMode: WaitForFirstConsumer <2>
 allowVolumeExpansion: true
 parameters:
```

Comment:
Azure Disk CSI driver has their parameters documented here: https://github.com/openshift/azure-disk-csi-driver/blob/release-4.14/docs/driver-parameters.md

`storageaccounttype` is deprecated, `skuName` should be used instead. And there is a plenty of new options. Again, not tested by us, so question is, if they should be documented.